doc/docs/master/how-tos/mount-volume.md

```diff
@@ -0,0 +1,184 @@
+# Mount a Volume in a Pipeline
+
+You may have a local or a network-attached storage that you want your
+pipeline to write files to. This way your data persists beyond the
+lifecycle of your pipeline.
```

Comment:
Data stored in PFS will also persist beyond the lifecycle of a pipeline, so this wouldn't be a motivating reason

Reply:
removed